docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -472,12 +472,15 @@ query brands {
 
 ## Complexity limits
 
-The GraphQL Storefront API uses an algorithm to calculate a complexity score for queries made against the API. Queries that exceed the complexity score will receive an error response:
+The GraphQL Storefront API uses an algorithm to calculate a complexity score for queries made against the API. The API returns the query complexity as an integer in the HTTP response header `x-bc-graphql-complexity` when you send a valid GraphQL request.
+
+If a query's complexity score exceeds the complexity limit, you will receive the following error response:
+
 
 ```json title="Example response with complexity error"
 {
   "error": {
-    "error": "The query is too complex as it has a complexity score of 1223 out of 1000. Please remove some elements and try again"
+    "error": "The query is too complex as it has a complexity score of 12230 out of 10000. Please remove some elements and try again"

```

Comment:
is there a period at the end of this sentence when you receive this message back? wondering if this is verbatim.

Reply:
No period, here's the screenshot https://github.com/bigcommerce/dev-docs/pull/1719#discussion_r1067456759